tutorials/jdbcIngest/src/site/conf/example.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<configuration>
+  <!-- <property> <name>dt.application.{appName}.operator.{opName}.prop.{propName}</name> 
+    <value>some-default-value (if value is not specified, it is required from 
+    the user or custom config when launching)</value> </property> -->
+  <!-- memory assigned to app master <property> <name>dt.attr.MASTER_MEMORY_MB</name> 
+    <value>1024</value> </property> -->
+
+  <!-- JDBC driver in use -->
+  <property>
+    <name>dt.application.SimpleJdbcToHDFSApp.operator.JdbcInput.prop.store.databaseDriver
+    </name>
+    <value>com.mysql.jdbc.Driver</value>
+  </property>
+
+  <!-- URL to connect to the DB master -->
+  <property>
+    <name>dt.application.SimpleJdbcToHDFSApp.operator.JdbcInput.prop.store.databaseUrl
+    </name>
+    <value>jdbc:mysql://node17.morado.com:5505/testDev</value>
```

Comment:
"node17.morado.com:5505" => "localhost:3306"
{3306 seems like the default port for MySql}